gazebo_plugins/src/gazebo_ros_skid_steer_drive.cpp

```diff
@@ -232,6 +252,16 @@ namespace gazebo {
     } else {
       this->update_period_ = 0.0;
     }
+
+    // gaussian noise
+    if (!_sdf->HasElement("gaussianNoise"))
+    {
+      ROS_DEBUG_NAMED("skid_steer_drive", "GazeboRosSkidSteerDrive Plugin (ns = %s) <gaussianNoise>, defaults to 0.0");
```

Comment:
there's a compiler warning here because the `%s` doesn't have a matching argument